src/SprykerEco/Zed/Heidelpay/Business/HeidelpayFacade.php

```diff
@@ -75,6 +93,36 @@ public function authorizePayment(OrderTransfer $orderTransfer): void
             ->authorize($orderTransfer);
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     *
+     */
+    public function authorizeOnRegistrationPayment(OrderTransfer $orderTransfer)
```

Comment:
And on facade methods. This should be done prior to release in order to not change a public api later